yarn-project/p2p/src/services/peer-manager/peer_manager.ts

```diff
@@ -190,10 +232,12 @@ export class PeerManager {
   private discover() {
     const connections = this.libP2PNode.getConnections();
 
-    const healthyConnections = this.prioritizePeers(this.pruneUnhealthyPeers(this.pruneDuplicatePeers(connections)));
+    const healthyConnections = this.prioritizePeers(
+      this.onlyNotTrustedPeers(this.pruneUnhealthyPeers(this.pruneDuplicatePeers(connections))),
```

Comment:
```suggestion
      this.pruneUnhealthyPeers(this.onlyNotTrustedPeers(this.pruneDuplicatePeers(connections))),
```

Reply:
reordering this means that the trusted peers arent included in the prune unhealthy filter